### PR TITLE
put request support

### PR DESCRIPTION
**Pull request by gaurav-unifydots**

### CODE REVIEW

File: src/test/resources/config/IN/QA/config.properties [hunk 1]
```diff
@@ -1 +1,5 @@
 base.url=https://www.saucedemo.com/
+apibase.url=https://reqres.in
+signup.url=/api/users
+update.url=/api/users/2
```
Comment (vivek-unifydots) [inline src/test/resources/config/IN/QA/config.properties:4]: userId should not be hardcoded

File: src/test/java/com/unifydots/apitests/SignUpAPITest.java [hunk 2]
```diff
@@ -29,5 +31,18 @@ public void signUpRequest() {
         signupDTO.setName(firstName);
         Response response = signupSteps.signup(signupDTO);
         Assert.assertEquals(response.statusCode(), HttpStatus.SC_CREATED);
+        String id= (String) APIUtils.getJsonElements(response.asString(), APIConstants.ID);
+        Assert.assertNotNull(id,"ID cannot be null");
+    }
+
+    @Test(description = "User wants to update")
+    public void UpdateRequest() {
```
Comment (vivek-unifydots) [inline src/test/java/com/unifydots/apitests/SignUpAPITest.java:39]: use camel case method names

File: src/test/resources/config/IN/QA/config.properties (hunk 1 above)
Comment (gaurav-unifydots) [reply src/test/resources/config/IN/QA/config.properties:4]: this api doesnot support path parameters.I tried to pass it but its throwing exception that path param is not expected

Comment (gaurav-unifydots) [reply src/test/resources/config/IN/QA/config.properties:4]: Api does not support path params

Comment (gaurav-unifydots) [reply src/test/resources/config/IN/QA/config.properties:4]: exception is coming that expected parameters are zero but provided one

**Review by vivek-unifydots — APPROVED** [review]

**Review by vivek-unifydots — APPROVED** [review]

**Review by vivek-unifydots — APPROVED** [review]

**Pull request merged**